workers/tests/unittests/test_custom_exceptions.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+from ohsome_quality_analyst.utils.exceptions import IndicatorLayerError
+
+
+class TestCustomException(unittest.TestCase):
+    def test(self):
+        with self.assertRaises(IndicatorLayerError):
+            raise IndicatorLayerError(("Indicator", "Layer"))
+        with self.assertRaises(ValueError):
+            raise IndicatorLayerError(("Indicator", "Layer"))
```

Comment:
I'm not 100% sure what you want to test here by using `("Indicator", "Layer")` but if you intend to give the Exception/Error more context, I would
1. add a description to the `IndicatorLayerError` definition, and
2. check these arguments in the tests as well, e.g. like this:
```suggestion
        indicator = "MyIndicator"
        layer = "MyLayer"
        with self.assertRaises(IndicatorLayerError) as cm:
            raise IndicatorLayerError((indicator, layer))
        self.assertEqual(cm.exception.args[0], (indicator, layer))
        with self.assertRaises(ValueError) as cm:
            raise IndicatorLayerError((indicator, layer))
        self.assertEqual(cm.exception.args[0], (indicator, layer))
```

Reply:
I added your suggestion to the tests.

Also I improved the exception message with a description + indicator layer tuple of validation function in base/indicator.py

Reply:
Okay, as far as I understand your code now, `IndicatorLayerError` should only take a string and not a tuple. I think you should change the tests then as well to check if the string matches (or get rid of the check again, as you only test the behaviour of `ValueError`).

Something like this:
```python
        with self.assertRaises(IndicatorLayerError) as cm:
            raise IndicatorLayerError("foo")
        self.assertEqual(cm.exception.args[0], "foo")

Reply:
Tuple would be fine. It would be cast to the string representation by the exception class. But you are right in that it is not needed here. I changed it to use a string as you suggested it

Reply:
Hmm okay, I didn't mean to get rid of the whole `IndicatorLayerError`, but okay ;)